pkg/collector/testdata/collector-override/override/input_usage.yaml

```diff
@@ -7,28 +7,28 @@ responses:
       containerID: app
       cpuCoresNanoSec: [1000000000, 2000000000, 3000000000, 2000000000]
       memoryBytes: [12000000, 18000000, 22000000, 26000000]
-      oomKillCount: [0, 0, 0, 0]
+      oomKillCount: 0
       cpuPeriodsSec: [100, 150, 200, 100]
       cpuThrottledPeriodsSec: [0, 2, 1, 2]
     - podUID: test-pod-1-deployment-1
       containerID: log-aggregator
       cpuCoresNanoSec: [2000000000, 3000000000, 4000000000, 2000000000]
       memoryBytes: [14000000, 20000000, 22000000, 24000000]
-      oomKillCount: [2, 2, 2, 2]
+      oomKillCount: 2
       cpuPeriodsSec: [400, 650, 500, 200]
       cpuThrottledPeriodsSec: [0, 9, 10, 20]
     - podUID: test-pod-1-deployment-2
       containerID: app
       cpuCoresNanoSec: [500000000, 1000000000, 1500000000, 2000000000]
       memoryBytes: [6000000, 9000000, 11000000, 13000000]
-      oomKillCount: [3, 3, 7, 8]
+      oomKillCount: 6
```

Comment:
This is a surprising change

Reply:
Since this is a total, it doesn't add much to have 300 1s samples, each incrementing.  The oomKillCount isn't going to vary wildly from second to second.  Its better to have a single sample.

Reply:
proto serialization should give us backwards compatibility using the field numbers.